torch/testing/_internal/opinfo/definitions/nested.py

```diff
@@ -106,6 +106,10 @@ def _slice_input(t, i=i, inp=nt_inp):
         args = tree_map(_slice_input, sample.args)
         kwargs = tree_map(_slice_input, sample.kwargs)
 
+        # Handle indices in index_put
+        if "index_put" in op.full_name and "indices" in kwargs:
+            kwargs["indices"] = [t[i] for t in kwargs["indices"][1:]]
```

Comment:
lol not too crazy about this - any way to avoid hard-coding the name here I wonder?

Reply:
this logic got even worse now, any suggestions?